pkg/agent/config/node_config.go

```diff
@@ -69,6 +78,9 @@ type NodeConfig struct {
 	PodCIDR *net.IPNet
 	// The Node's IP used in Kubernetes. It has the network mask information.
 	NodeIPAddr *net.IPNet
+	// Set either via defaultMTU config in antrea.yaml or auto discovered.
+	// Auto discovery will use mtu value of the node's primary interface (eth0).
```

Comment:
Please add comments to explain that for encap mode the mtu is using the value of the Node's primary interface substracting the tunnel header length.

Reply:
Done